### PR TITLE
Fix for #203

### PR DESCRIPTION
**Pull request by wdo3650**
Made the whole input field clickable. Fixed bug where the close button would reappear after clicking.

### CODE REVIEW

**Comment by philworthington** [comment]
The only issue here seems to be what we were saying in scrum earlier: There's a flicker when you click the close button. Also I noticed that the event for close button is 'mousedown'. Seems like it should just be a click or mouseup. Haven't tested it but this might contribute to the flicker.

File: src/components/SearchBox/SearchBox.less [hunk 1]
```diff
@@ -82,6 +89,7 @@
   display: none;
   font-size: @ms-font-size-l;
   color: @ms-color-white;
+  z-index: 2;
```
Comment (battletoilet) [inline src/components/SearchBox/SearchBox.less]: Use mixin @ms-zindex-front ?

File: src/components/SearchBox/Jquery.SearchBox.js [hunk 2]
```diff
@@ -31,24 +40,28 @@
...
+        }, 10);
+        
         /** Only do this if no text was entered. */
         if ($(this).val().length === 0 ) {
           $(this).siblings('.ms-SearchBox-label').show();
         }
 
         // Reset cancel to false
-        cancel = false;
+        cancel = false;
```
Comment (battletoilet) [inline src/components/SearchBox/Jquery.SearchBox.js]: Extra white space here.

**Comment by battletoilet** [comment]
APPPROOVEDZZZZ

**Pull request merged**